test/index.test.js

```diff
@@ -176,6 +176,28 @@ describe('Mixpanel', function() {
         analytics.called(window.mixpanel.people.set, { trait: true });
       });
 
+      it('should set super properties from the context.mixpanel.superProperties object if setAllTraitsByDefault is false', function() {
+        mixpanel.options.setAllTraitsByDefault = false;
+        analytics.identify(123, { accountStatus: 'Paid', subscribed: true }, { Mixpanel: { superProperties: ['accountStatus', 'subscribed'] } });
+        analytics.called(window.mixpanel.identify, 123);
+        analytics.called(window.mixpanel.register, { accountStatus: 'Paid', subscribed: true });
+      });
+
+      it('should set people properties from the context.mixpanel.peopleProperties object if setAllTraitsByDefault is false', function() {
+        mixpanel.options.people = false;
+        mixpanel.options.setAllTraitsByDefault = true;
+        analytics.identify(123, { friend: 'elmo' }, { Mixpanel: { peopleProperties: ['friend'] } });
+        analytics.called(window.mixpanel.identify, 123);
+        analytics.called(window.mixpanel.people.set, { friend: 'elmo' });
+      });
+
+      it('shouldn\'t set people properties from the context.mixpanel.peopleProperties object if setAllTraitsByDefault is false', function() {
+        mixpanel.options.people = false;
+        mixpanel.options.setAllTraitsByDefault = false;
+        analytics.identify(123, { friend: 'elmo' });
+        analytics.called(window.mixpanel.identify, 123);
```

Comment:
you're not testing that `people.set` or `register` **were not** called here, that's crucial.

something like
```
analytics.didNotCall('window.mixpanel.people.set`) 
```